guidelines/speaker-coordination.rst

```diff
@@ -0,0 +1,71 @@
+This document provides set of guidelines for coordinating Speakers for Workshop and Talks.
+
+
+Email guideline
+--------------
+
+- When contacting the Speaker or the participants, the Subject should be "[PYCON-INDIA-{enter year}][WORKSHOP]{reason of the email}".
+
+Todo before contacting the Speakers
+--------------------------
+
+- Call for volunteers, at least 4 volunteers per audi should be sufficient. This excludes the Lead Speaker Coordinator as (s)he will have to run between different audi.
+- Gather contact details of all the volunteers.
+- Do ask the volunteers if there is any specific Workshop/Talk they would like to attend, so that there won't be any missing volunteers at the last minute.
+
+Three weeks before the conference
+----------------------
+
+- Divide the volunteers into different audis and make a list of all speakers in the respective audis.
+- Inform the volunteers to contact the respective Workshop Speakers requesting if there is any addition to the requisite already mentioned in the Proposal.
+- The additional requisite can contain steps to install packages xyz, etc.
```

Comment:
This should be merged with the previous point. I think the below points would be a sensible addition:

- Inform speaker to get a flash drive with the necessary materials (installers, dependencies) for the talk.
- Ask volunteers to preferably get a few flash drives as well, for backup (optional).

What say @kracekumar ?